client/.vscodeignore

```diff
@@ -6,4 +6,7 @@ src/**
 **/*.map
 .gitignore
 tsconfig.json
-vsc-extension-quickstart.md
+tmp/**
+logs/**
+sessions/**
+gulpfile.js
```

Comment:
vscode ignore are for files to ignore when creating the extension.  I think you may be confusing this with gitignore.